src/stylesheets/main.css

```diff
@@ -305,6 +305,10 @@ footer a:hover {
   border-style: solid;
 }
 
+.scroll-link {

```

Comment:
This allows the anchor link to scroll an extra 240px (roughly double the height of the top nav), so that the anchor scrolls to here
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/722173f0-d5bd-4b87-a3d2-452a66ed230f">

instead of here:
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/9b3524de-230d-47a2-9cad-0767e5b322df">

In this scenario, the `Explore the intiatives` is hidden by the sticky nav.